applications/RANSApplication/custom_elements/data_containers/k_epsilon/epsilon_element_data.h

```diff
@@ -67,25 +86,18 @@ class EpsilonElementData : public ConvectionDiffusionReactionElementData
         const Matrix& rShapeFunctionDerivatives,
         const int Step = 0);
 
-    array_1d<double, 3> CalculateEffectiveVelocity(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+    ///@}
 
-    double CalculateEffectiveKinematicViscosity(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+protected:
+    ///@name Protected Members
+    ///@{
 
-    double CalculateReactionTerm(
-        const Vector& rShapeFunctions,
-        const Matrix& rShapeFunctionDerivatives) const;
+    using BaseType::mEffectiveVelocity;
+    using BaseType::mEffectiveKinematicViscosity;
+    using BaseType::mReactionTerm;
+    using BaseType::mSourceTerm;
```

Comment:
these are unnecessary; they're already `protected` in the base class.

Reply:
Since the base class is templated, I wanted to avoid using `this->*` allways. That is why i pu the `using *` here.

Reply:
aight